packages/dd-trace/test/debugger/devtools_client/utils.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const { randomUUID } = require('node:crypto')
+
+module.exports = {
```

Comment:
Usually, we write `module.exports` at the end of the file.

Reply:
enforced pattern I don't know tbh. It is what I see in other files, there is not a rule in the linter, so I think that it is not enforced pattern. It just catches my attention.